abn/application/application_test.go

```diff
@@ -76,7 +76,7 @@ func TestWrite(t *testing.T) {
 	a.Tracks["foo"] = "v1"
 
 	// Write writes immediately
-	Applications.Write(a)
+	_ = Applications.Write(a)
```

Comment:
Ignore errors in tests, if you do not use them in asserts.

Reply:
I gotta be honest, I find  it annoying to be explicit about this every time.

Reply:
Me too. But when it comes to code quality, as well as security, consistency is key. The linters make it reasonably easy to write high quality code in line with community standards. Of course, once we learn more about these linters, we can revisit how to do more fine grained configuration of these linters based on our tastes. For now, going with their suggestion seems very reasonable.